applications/portfolioDrafts/utils/validation.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Check whether a given string is valid JSON
+ *
+ * @param str - The String to check
+ * @returns true if JSON.parse is successful, otherwise returns false
+ */
+export function IsValidJson(str: string) {
```

Comment:
```suggestion
export function isValidJson(str: string) {
```

Reply:
Would be cool if this could just return a valid object when we got valid JSON. But I'd actually like to follow a model where our validators either return a valid object or they raise an exception (with `error_map` if needed).

Reply:
AT-6416 is meant to provide some guide posts around this, but I do agree that a model with raised exceptions which are caught and handled in a standardized way would be nice. Or at least, that's what I would do in Java.

Reply:
This should be included in a separate validation PR, we need to revamp how we are handling these errors.